parser/checkstyle.go

```diff
@@ -0,0 +1,73 @@
+package parser
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io"
+
+	"github.com/reviewdog/reviewdog/proto/rdf"
+)
+
+var _ Parser = &CheckStyleParser{}
+
+// CheckStyleParser is checkstyle parser.
+type CheckStyleParser struct{}
+
+// NewCheckStyleParser returns a new CheckStyleParser.
+func NewCheckStyleParser() Parser {
+	return &CheckStyleParser{}
+}
+
+func (p *CheckStyleParser) Parse(r io.Reader) ([]*rdf.Diagnostic, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method CheckStyleParser.Parse should have comment or be unexported